src/lib/project-saver-hoc.jsx

```diff
@@ -41,16 +43,39 @@ const ProjectSaverHOC = function (WrappedComponent) {
                     });
             }
             // TODO: distinguish between creating new, remixing, and saving as a copy
-            if (this.props.isCreating && !prevProps.isCreating) {
-                this.storeProject()
+            if (this.props.isCreatingNew && !prevProps.isCreatingNew) {
+                this.storeProject(null)
                     .then(response => {
                         this.props.onCreatedProject(response.id.toString(), this.props.loadingState);
                     })
                     .catch(err => {
-                        // NOTE: should throw up a notice for user
                         this.props.onProjectError(`Creating a new project failed with error: ${err}`);
                     });
             }
+            if (this.props.isCreatingCopy && !prevProps.isCreatingCopy) {
+                this.storeProject(null, {
+                    original_id: this.props.reduxProjectId,
+                    is_copy: 1
+                })
```

Comment:
Just noting what we already discussed, that we probably want to provide these to the `body` of our request, rather than configuring query params using storage.

Reply:
Doesn't this still need to be updated to send the params in the body rather than using the API for storage that we decided against?

Reply:
Weird, I could've sworn I had pushed after I rebased on top of develop yesterday. Sorry, please look again now.

Reply:
Oh — no this was my fault, I was misreading this as a call to storage, not our own helper function.